recitation-8/wordle_test.py

```diff
@@ -121,4 +121,9 @@ def test_gid_pizza():
 def test_gid_pizz():
     assert not(guess_in_dict('pizz', {'pizza': None}))
 
-# TODO write at least one test for `pick_word`
+
+def test_setup_anwser_animo():
+    dict_path = './words.json'
+    dictionary = load_words(dict_path)
+    wordle = WordleGame(dictionary, answer="animo")
+    assert wordle.answer_word == "animo"
```

Comment:
You are trying to test a word that you removed from the dictionary

Reply:
This looks like the problem to me, too